pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -256,6 +265,96 @@ func (vm *Windows) ConfigureCNI(configFile string) error {
 	return nil
 }
 
+// service represents a Windows service
+type service struct {
```

Comment:
Shouldn`t this be named as `kubeProxyService`. If we are planning to make a generic `service` object ,  we could have an interface with `start`, `remove` methods and then implement them

Reply:
I am unsure and would like further feedback from the rest of the team. My thought process was that each implementation of a possible interface would be doing the same thing with winRM. Thinking about it now, I'm not sure what makes more sense.

Reply:
My reasoning is also partly because I did not plan to have `service` be exposed outside of the package, and all calls to service objects would be wrapped with methods of Windows objects

Reply:
The `windows.go` file is growing. I suggest adding a `service.go`.